### DISCUSSION TITLE
Animated HEIC Files, part 3

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7770] [Help]
### ImageMagick version

7.1.1-40

### Operating system, version and so on

Windows 11

### Description

How do I use ImageMagick to make lossless heic/heif files? The pre-built installer versions contain libheif but it doesn't contain the x265 encoder. What should I do? For anyone who has experience with developer tools, how do I rebuild ImageMagick with a different configuration so that it has the encoder and the x265 library needed?

**Comment by dlemstra** [comment]
Please don't create a new discussion for the same question.
